src/chocolatey.resources/helpers/functions/Get-ChocolateyUnzip.ps1

```diff
@@ -77,6 +77,7 @@ param(
   [parameter(Mandatory=$true, Position=1)][string] $destination,
   [parameter(Mandatory=$false, Position=2)][string] $specificFolder,
   [parameter(Mandatory=$false, Position=3)][string] $packageName,
+  [parameter(Mandatory=$false, Position=4)][switch] $recursivecall,

```

Comment:
@ferventcoder - I don't have the bandwidth nor remember my test scenarios to update these.  I can, however, let you know what the thinking was behind each part.

In this case I followed the principle of keeping with the pattern of existing code - so inserted a position as the other's had.

I didn't know about documentation, but it is also an "internal" parameter that should only be called by the function itself under specific situations.